sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/models/SubscriptionProperties.java

```diff
@@ -189,7 +135,19 @@ public SubscriptionProperties setLockDuration(Duration lockDuration) {
      * @return the requiresSession value.
      */
     public boolean isSessionRequired() {
-        return this.requiresSession;
+        return toPrimitive(description.isRequiresSession());
+    }
+
+    /**
+     * Set the requiresSession property: A value that indicates whether the subscription supports the concept of
+     * sessions.
+     *
+     * @param requiresSession the requiresSession value to set.
+     * @return the SubscriptionDescription object itself.
+     */
+    public SubscriptionProperties setSessionRequired(boolean requiresSession) {
```

Comment:
@conniey @anuchandy this method was missing, do you know if it was intentional?

Reply:
Yes. We don't want this method exposed.

Reply:
Removed